src/openrct2/object/ObjectTypes.h

```diff
@@ -72,3 +72,9 @@ constexpr std::array<ObjectType, 16> TransientObjectTypes = {
     ObjectType::ParkEntrance, ObjectType::Water,        ObjectType::TerrainSurface,  ObjectType::TerrainEdge,
     ObjectType::Station,      ObjectType::Music,        ObjectType::FootpathSurface, ObjectType::FootpathRailings,
 };
+
+// Object types that cannot be saved in a park file.
+constexpr std::array<ObjectType, 2> IntransientObjectTypes = { ObjectType::ScenarioText, ObjectType::Audio };
```

Comment:
Since you have two functions to access the data its best this is moved into the C++ file, you can then also move the array include into the C++ file and save some compile time.

Reply:
Why did you mark this as resolved? That new global should be definitely moved into the cpp file, its only reference is those two functions so we don't need it in the header.

Reply:
As it brings it in alignment with the TransientObjectTypes

Reply:
> Why did you mark this as resolved? That new global should be definitely moved into the cpp file, its only reference is those two functions so we don't need it in the header.

That wasn't very clear, you crossed out the entire part that asked me to do something.

> As it brings it in alignment with the TransientObjectTypes

That would seem like a good reason to me as well. But now Matt and you are asking for opposing things - I will need you two to decide on one way or the other.

Reply:
Leave it as is I guess, that other stuff should be moved out of there eventually, so fine with me. Also if we have already a table that describes whats Transient then we don't need a separate table, its just the inverse of the type not being in the list.  Not asking to change it, given you added accessors this is trivial to change then.